modules/ROOT/pages/kubernetes/quickstart-analytics-cluster.adoc

```diff
@@ -0,0 +1,180 @@
+            path: gds.license
+
+# Set the environment variables to download the plugins:
+env:
+  NEO4J_PLUGINS: '["graph-data-science"]'
+
+# Set the configuration for the plugins directory and the mount for the license file:
+config:
+  gds.enterprise.license_file: "/licenses/gds.license"
+  server.directories.plugins: "plugins"
+
+# Disable the Neo4j load balancer and enable the internal service so that the servers can access each other:
+services:
+  neo4j:
+    enabled: false
+  internals:
+    enabled: true
+
+# Enable the analytics cluster and set the type to secondary:
+analytics:
+  enabled: true
+  type:
+    name: secondary
+
+----
+======
+=====
+
+For all available options, see xref:kubernetes/configuration.adoc[Customizing a Neo4j Helm chart].
+
+== Install the servers
+
+. Install a single Neo4j server using the _neo4j-primary.yaml_ file, created in the previous section:
++
+[source, bash]
+----
+helm install primary neo4j/neo4j -f /path/to/neo4j-primary.yaml
+----
+. Install the first secondary server using the _secondary-gds.yaml_ file, created in the previous section:
++
+[source, bash]
+----
+helm install gds1 neo4j/neo4j -f /path/to/secondary-gds.yaml
+----
+. Repeat step 2 to deploy a second secondary server.
+Use a different name, for example, _gds2_.
+
+== Verify that the GDS library is installed and licensed
+
+. Connect to each of the `gds` pods using the `kubectl exec` command:
++
+[source, bash]
+----
+kubectl exec -it gds1-0 -- bash
+----
+. From the _bin_ folder, connect to the `system` database of the `gds1` server using the `cypher-shell` command:
++
+[source, bash]
+----
+cypher-shell -u neo4j -p my-password -d system -a bolt://gds1-internals.default.svc.cluster.local:7687
+----
+. Run the following Cypher function to verify that the GDS library is installed:
++
+[source, cypher]
+----
+RETURN gds.version();
+----
+. Call `gds.isLicensed()` to verify that the GDS library is licensed:
++
+[source, cypher]
+----
+RETURN gds.isLicensed();
+----
++
+The returned value must be `true`.
+
+== Verify the cluster formation
+
+To verify that the cluster is deployed and running, you can install a load balancer and access Neo4j from the Neo4j Browser.
+
+. Deploy a Neo4j load balancer to the same namespace as the Neo4j cluster:
++
+[source, bash]
+----
+helm install lb neo4j/neo4j-load-balancer --set neo4j.name="analytics-cluster"
+----
+. When deployed, copy the `EXTERNAL_IP` of the LoadBalancer service.
+For more information, see xref:kubernetes/quickstart-cluster/access-outside-k8s.adoc[Access the Neo4j cluster from outside Kubernetes].
+. In a web browser, open the Neo4j Browser at _http://EXTERNAL_IP:7474/browser_ and log in using the password you have configured in your values YAML files.
+. Run the `SHOW SERVERS` command to verify that the cluster is deployed and running.
+. Run the `SHOW DATABASES` command to verify that primary server is of type `standalone` and the secondary servers are of type `gds`.
```

Comment:
shouldn't this be the other way around ? standalone is of type primary and gds of type secondary

Reply:
of course